src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationemployment/api/integration/resource/SARReadinessProfileGetShould.kt

```diff
@@ -66,17 +69,86 @@ class SARReadinessProfileGetShould : SARReadinessProfileTestCase() {
     assertAddReadinessProfileIsOk(prisonNumber, profileRequestOfKnownPRN)
     val expectedProfile = profileJsonOfKnownPRN
 
-    val sarResult = assertGetSARResponseIsOk(prn = prisonNumber)
+    val sarResult = assertGetSARResponseIsOk(expectedProfileAsJson = expectedProfile, prn = prisonNumber)
 
     assertThat(sarResult.body).isNotNull
     val json = objectMapper.readTree(sarResult.body!!.asJson())
     val jsonContent = json.findPath("content")
     assertThat(jsonContent.isMissingNode).isFalse()
     assertThat(jsonContent.get("offenderId").textValue()).isEqualTo(prisonNumber)
     assertThat(jsonContent.get("bookingId").longValue()).isEqualTo(bookingIdOfKnownPRN)
-    assertThat(jsonContent.get("profileData"))
-      .usingRecursiveComparison()
-      .ignoringFieldsMatchingRegexes(".*createdBy.*", ".*createdDateTime.*", ".*modifiedBy.*", ".*modifiedDateTime.*")
-      .isEqualTo(expectedProfile)
+  }
+
+  @Test
+  fun `reply 200(OK), when requesting a SAR with specified period`() {
+    val prisonNumber = knownPRN
```

Comment:
I prefer to avoid acronyms. For example, instead of `knownPRN` use `knownPrisonNumber`

Reply:
okay, and then the reader has to understand prisonNumber is PRN at SAR

Reply:
and I rename CRN too (also SAR term)

Reply:
updated